loculus_values/values.yaml

```
@@ -545,6 +569,9 @@ defaultOrganismConfig: &defaultOrganismConfig
         enableSubstringSearch: true
         columnWidth: 150
         order: 50
+        desired: true
+        definition: "List of author affiliations in the same order as authors, one entry per author"
```

We don't have one entry per author currently - so I think this should probably be removed. 
```suggestion
        definition: "List of author affiliations"
        guidance: "If multiple affiliations exist create a semi-colon separated list in the same order as authors with one entry per author "
```